### ISSUE TITLE
Missing lang attribute

### ISSUE THREAD

**Issue by rkaldung** [zammad/zammad#23]
Missing lang attribute for the html tag in the frontend markup. This is needed for screenreader and to support the browser to choose the proper quotes.
